README.md

```diff
@@ -6,10 +6,18 @@ This container image allows the user to provide their own script to run bootstra
 ## Using the Container Image
 
 For more information on settings, please refer to the [Bottlerocket Documentation](https://bottlerocket.dev/) in order to configure your files. 
-To use the container image, create a TOML file with the following configurations. 
+The built-in bootstrap container source will be used by default, if source is not specified in the configuration.
```

Comment:
I'm not sure built-in is the right word since we do pull it at boot, we just automatically update it. So maybe "The automatically updated bootstrap container..."